wp-content/mu-plugins/register-rest-routes.php

```diff
@@ -288,3 +290,187 @@
     }
   ));
 });
+
+add_action('rest_api_init', function() {
+  $v = 'api/v1'; // namespace for the current version of the API
+
+  $exp = WEEK_IN_SECONDS; // expiration of the transient caches
+  
+
+  register_rest_route('api/v1', '/confirmation-events/', array(
+    'methods' => 'POST',
+    'permission_callback' => '__return_true',
+
+    'callback' => function(WP_REST_Request $request) {
+
+      $path = ABSPATH . 'wp-content/mu-plugins/config/config.yml';
+
+      $sendgrid_api_key;
+      $sendgrid_confirmation_template_id;
+      $sendgrid_confirmed_template_id;
+      $sendgrid_contact_list_id;
+     
+      //Fetch the Sendgrid Configuration values from config file
+       if (file_exists($path)) {
```

Comment:
- [ ] You should be able to remove this and use the environment variables directly. See the Readme file here: https://github.com/CityOfNewYork/working-nyc/tree/main/wp-content/mu-plugins/wp-config
- [ ] Add all new config variables to config-sample.yml